contracts/utils/RateLimited.sol

```diff
@@ -85,6 +85,11 @@ abstract contract RateLimited is CoreRef {
             Math.min(_bufferStored + (rateLimitPerSecond * elapsed), bufferCap);
     }
 
+    /// @notice return buffer stored to validate state transitions
```

Comment:
Why not just make the _bufferStored var public?

Reply:
that's a really good idea